### PR TITLE
Streamline tagging

### PR DESCRIPTION
**Pull request by giorgiosironi**
Each image is tagged with a date, and overwrites `:latest`

### CODE REVIEW

**Comment by giorgiosironi** [comment]
I'll add a pipeline here, may be heavy to rebuild all images but these are base images for everything else, hence very important to smoke test.

File: Jenkinsfile.php_fpm [hunk 1]
```diff
@@ -10,19 +10,18 @@ elifePipeline {
...
             stage 'Build image', {
-                checkout scm
-                sh 'docker build -f Dockerfile.php_fpm -t elifesciences/php_fpm .'
+                sh "docker build -f Dockerfile.php_fpm -t elifesciences/php_fpm:${tag} ."
             }
 
             stage 'Sanity tests', {
-                sh 'docker run -it elifesciences/php_fpm /usr/bin/env php -r "echo 42, PHP_EOL;"'
-                sh 'docker run -it elifesciences/php_fpm composer -V'
+                sh "docker run -it elifesciences/php_fpm:${tag} /usr/bin/env php -r "echo 42, PHP_EOL;""
```
Comment (thewilkybarkid) [inline Jenkinsfile.php_fpm]: Needs single quotes?

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        {
+            def commit
+            stage 'Checkout', {
+                checkout scm
+                commit = elifeGitRevision()
+            }
+
+            stage 'Build images', {
+                def actions = [:]
+                def images = ['php_cli', 'php_fpm']
```
Comment (thewilkybarkid) [inline Jenkinsfile]: Probably not worth file scanning.

Comment (giorgiosironi) [reply Jenkinsfile]: Not yet

**Pull request merged**